core/App/lib_jsoncpp/json/autolink.h

```diff
@@ -0,0 +1,19 @@
+#ifndef JSON_AUTOLINK_H_INCLUDED
```

Comment:
where's the lib_jsoncpp comes from?

you may not put the codes under ehsm/core/app folder, change it to:
ehsm/core/utils/lib_jsoncpp

Reply:
then add a readme under that folder to descript to source destination? is there a git link?

Reply:
why not install libjson-c-dev  libjsoncpp-dev in Dockerfile?
Is your lib_jsoncpp same as apt install libjsonc-dev?

Reply:
yes, if we can directly use the apt install, it's more better.

Reply:
The jsoncpp will be remove.